src/state/account.ts

```diff
@@ -86,6 +88,16 @@ export const accountSlice = createSlice({
     setTransfers: (state, action: PayloadAction<any>) => {
       state.transfers = action.payload;
     },
+    setLatestOrder: (state, action: PayloadAction<Nullable<SubaccountOrder>>) => {
```

Comment:
@rosepuppy When Abacus updates latestOrder we also remove the `clientId` from the `uncomittedOrderClientIds`.